src/widget/drop_down_list.rs

```diff
@@ -280,7 +282,7 @@ impl<'a, F> Widget for DropDownList<'a, F> where
 
                     MenuState::Closed
                 // Otherwise if the mouse was released somewhere else we should close the menu.
-                } else if global_mouse.left.was_just_pressed
+            } else if global_mouse.buttons.get(MouseButton::Left).was_just_pressed
```

Comment:
Seeing as this is the most common case, perhaps there could be a `left` helper method for easier access that doesn't require importing `MouseButton`, i.e. this would allow `global_mouse.buttons.left().was_just_pressed`.